python/pyproject.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 requires-python = ">=3.7"
 dependencies = [
     "requests >= 2.28.1",
-    "websocket-client >= 1.4.2"
+    "python-socketio >= 5.11.0"
```

Comment:
From @JasperTan97's testing:
```suggestion
    "python-socketio[client] >= 5.11.0"
```

Reply:
Don't forget to commit the suggestion!

Reply:
yes indeed. is it fine with you if I don't merge this PR until we actually release v3?